docs/ibexa_products/ibexa_commerce.md

```diff
@@ -0,0 +1,96 @@
+
+![Ibexa Commerce](ibexa_commerce.png)
+
+## Availability
+
+To start using [[= product_name_com =]] you need to purchase a product license.
+For more information, see [[[= product_name_com =]] license pricing](https://www.ibexa.co/products/pricing?tab=3).
+You can also [contact us](https://www.ibexa.co/about-ibexa/contact-us) or [one of our partners](https://www.ibexa.co/partners).
+
+## How it works
+
+### Technical backstage
+
+With an active lincese, you can start [installation process](install_ibexa_dxp.md) that uses the Composer.
+
+[[= product_name_com =]] is based on [Symfony]([[= symfony_doc =]]). With a help of documentation and trainings, any developer familiar with Symfony or simply PHP may learn how to use available extension points and extend the platform.
+
+Version control systems and environment variables allow you to deploy your extensions and settings on several environments, such as [Ibexa Cloud](ibexa_cloud_guide.md).
+
+[[= product_name_com =]] is built on [[[= product_name_headless =]]](headless.md) and includes all bundles, APIs, and [features that comes with Headless edition](headless.md#core-features) as well as [Experience features](experience.md#core-features).
+
+## Capabilities and benefits
+
+[[= product_name_com =]] includes the greatest capabilities to accelerate your digital transformation: Personalized workflows, corporate account management, customized catalogs, and more.
+
+### Core features
+
+[[= product_name_com =]] includes all the features you need to launch your online store and reduce the time it takes to go live.
+
+#### Order management
+
+With the advanced [Order management]([[= user_doc =]]/commerce/order_management/order_management/) tools, you can easily manage your customized purchase process.
+The Order management module allows users to review order details and updates, track order completion status, and cancel orders that are created when store customers purchase products, for a smooth purchasing process.
+You can also assign and maintain rights so that only authorized users and make modifications.
+When searching for orders you can use filters, so you save time and reduce effort.
+
+![Order management](order_management.png)
+
+#### Merchandizing
+
+Merchandizing assists in keeping the consistency of the brand and providing customized product recommendations with captivating visuals and powerful search features. 
+With this feature you can engage your customers with eye-catching graphics and information
+The search engine makes it easy to find what you're looking for by providing quick and easy access to the product catalogs.
+You can also use recommendations and campaigns to increase average order values and income.
+
+#### eCommerce
+
+With the [eCommerce feature]([[= user_doc =]]/personalization/use_cases/#ecommerce) you can transform your online stores and give your consumers great buying experiences, from initial contact to post-purchase support.
```

Comment:
```suggestion
#### Product recommendations

Use [Ibexa Personzalition]([[= user_doc =]]/personalization/use_cases/#ecommerce) to transform your online stores and give your consumers great buying experiences, from initial contact to post-purchase support.
```
I wouldn't use the name "eCommerce feature" - here you're describing the Personalization use case for eCommerce.